src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -279,15 +281,15 @@ private BibEntry cleanup(BibEntry entry) {
             title = title.replaceAll("/[sS]pl ([^/]+)/", "\\$\\\\$1\\$");
             // Deal with subscripts and superscripts
             if (Globals.prefs.getBoolean(JabRefPreferences.USE_CONVERT_TO_EQUATION)) {
-                title = title.replaceAll("/sup ([^/]+)/", "\\$\\^\\{$1\\}\\$");
-                title = title.replaceAll("/sub ([^/]+)/", "\\$_\\{$1\\}\\$");
-                title = title.replaceAll("\\(sup\\)([^(]+)\\(/sup\\)", "\\$\\^\\{$1\\}\\$");
-                title = title.replaceAll("\\(sub\\)([^(]+)\\(/sub\\)", "\\_\\{$1\\}\\$");
+                title = title.replaceAll(SUPER_DETECTION_1, SUPER_EQ_RESULT);
+                title = title.replaceAll(SUB_DETECTION_1, SUB_EQ_RESULT);
+                title = title.replaceAll(SUPER_DETECTION_2, SUPER_EQ_RESULT);
+                title = title.replaceAll(SUB_DETECTION_2, SUB_EQ_RESULT);
```

Comment:
For performance reasons, you should create Patterns as static variables and then use them to perform the replace all.